test/dynamo/test_functions.py

```diff
@@ -3306,6 +3306,46 @@ def fn(x):
 
         self.assertEqual(fn(z), fn_opt(z))
 
+    def test_add_global_set(self):
+        # This fail with enable_cpp_guard_manager = True.
+        torch._dynamo.config.enable_cpp_guard_manager = False
```

Comment:
Use `config.patch` otherwise this will change the behavior for tests that run after thos one.

Reply:
i will remove this after rebase on https://github.com/pytorch/pytorch/pull/134016